### ISSUE TITLE
No actual toolchain_type for @io_bazel_rules_rust//rust:toolchain

### ISSUE THREAD

**Issue by katre** [bazelbuild/rules_rust#128]
With the fixes for bazelbuild/bazel#6015, there will be an actual dependency from toolchains back to the respective toolchain_type target. rules_rust currently does not actually define the toolchain_type, and so builds fail (see https://buildkite.com/bazel/bazel-with-downstream-projects-bazel/builds/480#f43a9298-b4e0-46da-980b-e6bcacdd5e5c).

Error:
```
ERROR: /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/7a687a53d9d341692c7f46223f63d5ca/external/rust_linux_x86_64_toolchains/BUILD:2:1: no such target '@io_bazel_rules_rust//rust:toolchain': target 'toolchain' not declared in package 'rust' (did you mean 'toolchain.bzl'?) defined by /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/7a687a53d9d341692c7f46223f63d5ca/external/io_bazel_rules_rust/rust/BUILD and referenced by '@rust_linux_x86_64_toolchains//:toolchain_for_x86_64-unknown-linux-gnu'
```

**Comment by katre** [comment]
The fix is to define the toolchain_type target, which I will send a PR for shortly.

How difficult will it be to get this into a released rules_rust before Bazel 0.20.0 is cut (early November)?

**Comment by acmcarther** [comment]
Released [0.0.7](https://github.com/bazelbuild/rules_rust/releases/tag/0.0.7)
